payroll/admin.py

```diff
@@ -55,8 +63,51 @@ def most_recent_job(self, obj):
         return obj.most_recent_job
 
 
+class PayrollSettingAdmin(SettingAdmin):
+    def save_model(self, request, obj, form, change):
+        super().save_model(request, obj, form, change)
+
+        # Remove the homepage from the cache if the donate banner value has
+        # changed.
+        if obj.name == 'PAYROLL_SHOW_DONATE_BANNER' and change:
+            try:
```

Comment:
hi hannah, could you provide some context about why you didn't use higher level methods like

[get_cache_key](https://github.com/django/django/blob/12d6cae7c0401baa70c934f465bad856afecc847/django/utils/cache.py#L345)

and `cache.delete`

Reply:
`get_cache_key` expects a request object for the index page. Don't have a good way to get a handle on that – tried making one in https://github.com/datamade/bga-payroll/pull/493/commits/357591ae9e1b1294beb6fa2cdd4e126ea4b49b21, which worked locally but not in deployment environemnts. (Shelled in and tried a variety of hosts / ports, none of which generated the key I needed.)

Also, Django caches the headers and request, so there are two cache entries associated with the index. 

Looking at the components of both header / request cache keys, it seemed simplest to replicate the hash for the index URL and remove any keys containing it from the cache in SQL, instead of trying to build out both keys.